[2025-03-10T08:12:01.743+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-10T08:12:03.114+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.customers manual__2025-03-10T08:11:42.133144+00:00 [queued]>
[2025-03-10T08:12:03.250+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.customers manual__2025-03-10T08:11:42.133144+00:00 [queued]>
[2025-03-10T08:12:03.277+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-10T08:12:03.369+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): customers> on 2025-03-10 08:11:42.133144+00:00
[2025-03-10T08:12:03.414+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4200) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-10T08:12:03.429+0000] {standard_task_runner.py:72} INFO - Started process 4217 to run task
[2025-03-10T08:12:03.443+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'customers', 'manual__2025-03-10T08:11:42.133144+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/validate_all_tables_dag.py', '--cfg-path', '/tmp/tmphwtci325']
[2025-03-10T08:12:03.489+0000] {standard_task_runner.py:105} INFO - Job 139: Subtask customers
[2025-03-10T08:12:04.050+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.customers manual__2025-03-10T08:11:42.133144+00:00 [running]> on host efacfdd0be48
[2025-03-10T08:12:05.428+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='customers' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T08:11:42.133144+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T08:11:42.133144+00:00'
[2025-03-10T08:12:05.431+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-10T08:12:05.449+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-10T08:12:05.474+0000] {logging_mixin.py:190} INFO - Current task name:customers state:running start_date:2025-03-10 08:12:03.151141+00:00
[2025-03-10T08:12:05.476+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-10T08:12:05.491+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-10T08:12:06.396+0000] {logging_mixin.py:190} INFO - Customers validation and cleaning successful
[2025-03-10T08:12:06.397+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-10T08:12:06.552+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-10T08:12:06.566+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=customers, run_id=manual__2025-03-10T08:11:42.133144+00:00, execution_date=20250310T081142, start_date=20250310T081203, end_date=20250310T081206
[2025-03-10T08:12:06.936+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-10T08:12:06.978+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-10T08:12:06.991+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-10 08:11:42.167048+00:00
[2025-03-10T08:12:07.007+0000] {logging_mixin.py:190} INFO - Task hostname:efacfdd0be48 operator:PythonOperator
[2025-03-10T08:12:07.127+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-10T08:12:07.511+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-10T08:12:07.541+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
